### ISSUE TITLE
Minimum altitude is not necessarily the first point

### ISSUE THREAD

**Issue by miqwit** [Sibyx/phpGPX#70]
https://github.com/Sibyx/phpGPX/blob/ecde89a9268f78340fa9c016ff68115d3efa9a42/src/phpGPX/Models/Route.php#L102

In some sports or activities, including hiking, you could start from a higher point than the one you finish. Using the ElevationGainLossCalculator could return the lowest elevation in the points without considering the first one necessarily.

**Comment by Sibyx** [comment]
Yeah, makes sense. Would you like to have it resolved in 1.x or 2.x?

**Comment by miqwit** [comment]
I have no idea, really....  both? :)

**Comment by Sibyx** [comment]
@miqwit yeah, that would be a best case :D
